src/cases/containers/case-share-complete/case-share-complete.component.ts

```diff
@@ -19,8 +20,9 @@ export class CaseShareCompleteComponent implements OnInit, OnDestroy {
   public shareCaseState$: Observable<fromCasesFeature.ShareCasesState>;
   public isLoading: boolean;
   public completeScreenMode: string;
+  public removeUserFromCaseToggleOn$: Observable<boolean>;
 
-  constructor(public store: Store<fromCaseList.State>) {}
+  constructor(public store: Store<fromCaseList.State> , public featureToggleService: FeatureToggleService) {}
```

Comment:
Why are the parameters `public` and not `readonly`? They don't seem to be being accessed by anything _other_ than this component so they should be okay to be `private readonly`.

Reply:
This is for the unit tests, will fail if change to private

Reply:
You're _providing_ them within the unit tests so you already have a reference to the `store` and the `featureToggleService`. They don't need to be `public` on the component in order for the tests to work as you never access `component.store` or `component.featureToggleService` within the tests.

Reply:
done